crates/holochain/tests/tests/countersigning.rs

```diff
@@ -130,6 +136,186 @@ async fn listen_for_countersigning_completion() {
     wait_for_completion(bob_rx, preflight_request.app_entry_hash).await;
 }
 
+// Regression test to check that it's possible to continue a countersigning session following
+// a failed commit that required dependencies that couldn't be fetched.
+#[tokio::test(flavor = "multi_thread")]
+async fn retry_countersigning_commit_on_missing_deps() {
```

Comment:
⭐

Reply:
A gold star! Gotta print this out and pin it on me :)

Reply:
I can send you one via mail as well if you like. Here's another one 🌟